python_ta/typecheck/typeshed/builtins.pyi

```diff
@@ -391,7 +391,7 @@ class bytes(ByteString):
     def __rmul__(self, n: int) -> bytes: ...
     if sys.version_info >= (3, 5):
         def __mod__(self, value: Any) -> bytes: ...
-    def __contains__(self, o: object) -> bool: ...
+    def __contains__(self, o: bytes) -> bool: ...
```

Comment:
On the other hand, this (and likely the others down below) is too strict:

```python
>>> 3 in b'123'
False
```
I don't mind making the type-checker more strict here, but you should articulate this in the README as well.